.gitattributes

```diff
@@ -0,0 +1 @@
+* text eol=lf
```

Comment:
Looking at the docs, this setting doesn't support the special value `input`, doesn't it?

Your problem was automatic conversion to CRLF, which was wrong, but keeping things as is would have prevented that problem as well. That's exactly what `autocrlf = input` does in the end and what I configure by default system wide. That's why things succeeded for me. LF stay LF, CRLF stay CRLF, LF will NEVER be CRLF unless one converts manually for some reason.

The downside of setting LF is that CRLF of the repo becomes LF in the working copy even if CRLF would be correct and needed to be changed manually, e.g. because tests would needed to be adopted etc. So if supported, `input` would be a better choice in my opinion. Just would like to make sure it isn't supported.

Reply:
It doesn't look like it, but we could perhaps be a bit more specific.  I think we could do something like
```
src/test/witness/* -text
```
to tell git to not normalize the line endings, and then having a `* text=auto` entry for everything else(I haven't tried this out).  I don't view it as a big deal, since most(all?) IDEs should take care of that, although it's possible that we could get into a situation where \r\n is mixed with \n in the same file.

Reply:
If we agree that `input` is not supported, keep things as they are. No need to make things unnecessary complex and `auto` pretty much always is a bad choice in my opinion. If someone needs to introduce CRLF in some test by purpose, this can be reconsidered.